### ISSUE TITLE
"Remove emoji from user names" should maybe remove more characters

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/pinafore#1805]
Got a feature request recently: some user names contain "latin small capital" and "dingbat negative sans-serif" which are both annoying for screen reader users. It should be possible to filter these in addition to native and custom emoji.
